src/components/TodoList.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { User } from '../types/User';
+import { Todo } from '../types/Todo';
+
+import './TodoList.scss';
+// import todos from '../api/todos';
+
+type Props = {
+  todos: Todo[];
+  users: User[];
+};
+
+const formatPhoneNumber = (user: User) => (
+  user.phone.split('').filter(el => !Number.isNaN(+el)).slice(0, 11).join('')
+);
+
+const getVisibleNumber = (str: string) => (
+  str.length > 10
+    ? `+${str[0]} (${str.slice(1, 4)}) ${str.slice(4, 7)} ${str.slice(7)}`
+    : `(${str.slice(0, 3)}) ${str.slice(3, 6)} ${str.slice(6)}`
+);
+
+export class TodoList extends React.PureComponent<Props> {
+  render() {
+    const { todos, users } = this.props;
+    const preparedTodos = todos.map((todo) => ({
+      ...todo,
+      user: users.find(user => user.id === todo.userId),
+    }));
```

Comment:
do this in App.tsx (but outside the component itself) and pass to TodoList just preparedTodos